code/modules/mob/living/carbon/human/human_defines_vr.dm

```diff
@@ -6,4 +6,5 @@
 	var/wagging = 0 //UGH.
 	var/flapping = 0
 	var/vantag_pref = VANTAG_NONE //What's my status?
-	var/impersonate_bodytype //For impersonating a bodytype
\ No newline at end of file
+	var/impersonate_bodytype //For impersonating a bodytype
+	var/fruit_type = "apple" // Decides the frruit type. Not used outside of Alraunes
```

Comment:
Not on every human either :C This is the opposite of good object-oriented programming